state/benches/storage.rs

```diff
@@ -0,0 +1,142 @@
+use common::types::{wallet::Wallet, wallet_mocks::mock_empty_wallet};
+use criterion::{black_box, criterion_group, criterion_main, BenchmarkId, Criterion, Throughput};
+use rand::{thread_rng, Rng};
+use state::{storage::db::DB, test_helpers::mock_db};
+
+// -----------
+// | Helpers |
+// -----------
+
+/// A table used for benchmarking
+const BENCHMARK_TABLE: &str = "benchmark_table";
+
+/// One byte
+const ONE_BYTE: usize = 1;
+/// One kilobyte
+const ONE_KILOBYTE: usize = 1024;
+/// One megabyte
+const ONE_MEGABYTE: usize = 1024 * ONE_KILOBYTE;
+
+/// Create a mock database for testing
+fn benchmark_db() -> DB {
+    let db = mock_db();
+    db.create_table(BENCHMARK_TABLE).unwrap();
```

Comment:
not necessary rn, but just crossed my mind that we could play around w/ various levels of fill in the db for benches